diff --git a/dist/index.mjs b/dist/index.mjs
index 4ad258e0db216750eb3b1d231fdf0923049221ee..b1b26521b183b7e22e89117d85e22a8499da262b 100644
--- a/dist/index.mjs
+++ b/dist/index.mjs
@@ -1,12 +1,12 @@
 import { InvalidProtocolSchemeError, FailEntirely, FailFallible, SucceedEntirely } from '@midnight-ntwrk/midnight-js-types';
 import { Transaction, ZswapChainState } from '@midnight-ntwrk/ledger';
 import { ContractState } from '@midnight-ntwrk/compact-runtime';
-import { ApolloClient, InMemoryCache } from '@apollo/client/core/core.cjs';
-import { from, split } from '@apollo/client/link/core/core.cjs';
-import { createHttpLink } from '@apollo/client/link/http/http.cjs';
-import { RetryLink } from '@apollo/client/link/retry/retry.cjs';
-import { getMainDefinition } from '@apollo/client/utilities/utilities.cjs';
-import { GraphQLWsLink } from '@apollo/client/link/subscriptions/subscriptions.cjs';
+import { ApolloClient, InMemoryCache } from '@apollo/client/core';
+import { from, split } from '@apollo/client/link/core';
+import { createHttpLink } from '@apollo/client/link/http';
+import { RetryLink } from '@apollo/client/link/retry';
+import { getMainDefinition } from '@apollo/client/utilities';
+import { GraphQLWsLink } from '@apollo/client/link/subscriptions';
 import { Buffer } from 'buffer';
 import * as Rx from 'rxjs';
 import fetch from 'cross-fetch';
